prometheus/config/prometheus.yml

```diff
@@ -0,0 +1,29 @@
+global:
+  scrape_interval: 1m
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:9090']
+  - job_name: json_exporter
+    static_configs:
+      - targets: ['json-exporter:7979']
+  - job_name: grafana
+    static_configs:
+      - targets: ['grafana:3000']
+  - job_name: cht
+    scrape_interval: 5m
+    metrics_path: /probe
+    params:
+      module: [default]
+    file_sd_configs:
+      - files:
+        - cht-instances.yml
+    relabel_configs:
```

Comment:
Right now the tool tips when you hover are quite long, but I don't think they should block MVP UNLESS it will be super invasive/breaking change-y to fix.  

For example, the first two panels I hover over are great - exactly as expected. The second two are LOOOONG strings that don't make sense and cause confusion as to what I'm looking at:

https://user-images.githubusercontent.com/8253488/231599334-0575043c-4b24-4d24-8f44-32f2a3b74f9b.mp4

Reply:
Honestly, this was an easy fix. So I just pushed the changes now while I was thinking about it.  

Grafana logic for the tool tips is a bit weird, and so not all of them are totally consistent (some are stuck showing the title of the panel). For the ones with really long JSON names, I have updated them to just be the name of the metric being displayed.  

These will be easy to update/evolve in future iterations.